clusterloader2/testing/experimental/storage/pod-startup/config.yaml

```diff
@@ -1,96 +1,151 @@
 # - Number of nodes should be divisible by NODES_PER_NAMESPACE (default 100).
 
-#Constants
-# TODO(mucahitkurt) copied from load tests, open for discussion.
-{{$NODE_MODE := DefaultParam .NODE_MODE "allnodes"}}
-{{$NODES_PER_NAMESPACE := DefaultParam .NODES_PER_NAMESPACE 100}}
-{{$PODS_PER_NODE := DefaultParam .PODS_PER_NODE 30}}
+# Cluster Variables
+  {{$NODE_MODE := DefaultParam .NODE_MODE "allnodes"}}
+  {{$NODES_PER_NAMESPACE := DefaultParam .NODES_PER_NAMESPACE 100}}
 
-{{$POD_TEMPLATE_PATH := DefaultParam .POD_TEMPLATE_PATH "../volume-types/emptydir/pod_with_emptydir.yaml"}}
-{{$LOAD_TEST_THROUGHPUT := DefaultParam .LOAD_TEST_THROUGHPUT 10}}
+# Framework Variables
+  {{$OBJ_CREATION_QPS := DefaultParam .OBJ_CREATION_QPS 10}}
+  {{$PARALLELISM_LIMITED := DefaultParam .PARALLELISM_LIMITED 1}}
+  {{$ENABLE_CHAOSMONKEY := DefaultParam .ENABLE_CHAOSMONKEY false}}
+  {{$TEST_NAME := DefaultParam .TEST_NAME "generic-test"}}
+  {{$GROUP := "pod-with-volumes-startup-latency"}}
+  {{$TUNING_SET := DefaultParam .TUNING_SET "UniformQPS"}}
 
-{{$VOLUMES_PER_POD := DefaultParam .VOLUMES_PER_POD 1}}
-{{$VOLUME_TEMPLATE_PATH := .VOLUME_TEMPLATE_PATH}}
-{{$PROVISION_VOLUME := DefaultParam .PROVISION_VOLUME false}}
+# Test Variales
+  {{$PODS_PER_NODE := .PODS_PER_NODE}}
+  {{$POD_TEMPLATE_PATH := .POD_TEMPLATE_PATH }}
+  {{$VOLUMES_PER_POD := .VOLUMES_PER_POD}}
+  {{$VOLUME_TEMPLATE_PATH := .VOLUME_TEMPLATE_PATH}}
+  {{$PROVISION_VOLUME := DefaultParam .PROVISION_VOLUME false}}
+  {{$VOL_SIZE := DefaultParam .VOL_SIZE "8Gi"}}
+  {{$WAIT_FOR_PVC := DefaultParam .WAIT_FOR_PVC false}}
+  {{$POD_STARTUP_TIMEOUT := DefaultParam .POD_STARTUP_TIMEOUT "15m"}}
 
-{{$APP_NAME := "pod-load"}}
-{{$GROUP := "pod-with-ephemeral-volume-startup-latency"}}
+# SLO Variables
+  {{$POD_STARTUP_SLO := DefaultParam .POD_STARTUP_SLO 300}}
+  {{$PER_PVC_BOUND_SLO := DefaultParam .PVC_BOUND_SLO 2}}
 
-#Variables
-# used stateless pod SLO for initial run
-{{$namespaces := DivideInt .Nodes $NODES_PER_NAMESPACE | MaxInt 1}}
-{{$podStartupTimeout := 5}}
-{{$totalPods := MultiplyInt $namespaces $NODES_PER_NAMESPACE $PODS_PER_NODE}}
-{{$podsPerNamespace := MultiplyInt $NODES_PER_NAMESPACE $PODS_PER_NODE}}
-{{$volumesPerNamespace := MultiplyInt $podsPerNamespace $VOLUMES_PER_POD}}
+# Computed Variables
+  {{$namespaces := DivideInt .Nodes $NODES_PER_NAMESPACE | MaxInt 1}}
+  {{$totalPods := MultiplyInt $namespaces $NODES_PER_NAMESPACE $PODS_PER_NODE}}
+  {{$podsPerNamespace := MultiplyInt $NODES_PER_NAMESPACE $PODS_PER_NODE}}
+  {{$volumesPerNamespace := MultiplyInt $podsPerNamespace $VOLUMES_PER_POD}}
+  {{$totalVols := MultiplyInt $volumesPerNamespace $namespaces}}
+  {{$PVCBoundTime := MultiplyInt $totalVols $PER_PVC_BOUND_SLO | MaxInt 60}}
 
-name: pod-with-ephemeral-volume-startup-latency
+
+name: pv-storage-{{$TEST_NAME}}-{{$VOLUME_TYPE}}
 automanagedNamespaces: {{$namespaces}}
 tuningSets:
-- name: UniformQPS
-  qpsLoad:
-    qps: {{$LOAD_TEST_THROUGHPUT}}
+  - name: Sequence
```

Comment:
This isn't used anywhere - let's remove it.
Allowing to many knobs to configure is never good - the more opinionated we are the better.

Reply:
Ok, I defaulted it to sequence with PL=1.